client/state/domains/nameservers/selectors.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Internal dependencies
+ */
+import { initialDomainState } from './reducer';
```

Comment:
Having the `initialDomainState` in the `reducer` module makes the entire reducer part of any chunk that uses the selector.

It probably doesn't matter much today when we have dynamic reducers and the selector and reducer code is no longer strictly decoupled from each other.

But having the initial state in a separate module (`./initial`) still might be more friendly for the bundle.

Reply:
Sure - addressed in cedf05e19eeceed9088ed628a0d906ab6a8dd4b4